### PR TITLE
Git#2 homework done

### PR DESCRIPTION
**Pull request by MechaniQ1**

### CODE REVIEW

File: src/functions.cpp [hunk 1]
```diff
@@ -1,26 +1,30 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include "functions.hpp"  // TODO: add proper compilation flag and change to #include "functions.hpp"
```
Comment (DamiXioR) [inline src/functions.cpp]: remove comments

File: src/functions.cpp [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
+
 #include <algorithm>
 #include <iostream>
 #include <limits>
 
-void print(std::vector<int> const & numbers) {
+void print(std::vector<int> const& numbers) {
     for (int i = 0; i < numbers.size(); ++i) {
```
Comment (DamiXioR) [inline src/functions.cpp]: use std::size_t

File: src/main.cpp [hunk 3]
```diff
@@ -1,6 +1,7 @@
-#include <vector>
 #include <iostream>
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include <vector>
+
+#include "functions.hpp"  // TODO: add proper compilation flag and change to #include "functions.hpp"
```
Comment (DamiXioR) [inline src/main.cpp]: remove comments

File: Makefile [hunk 4]
```diff
@@ -1,3 +1,3 @@
 all: src/*.cpp inc/*.hpp
-	g++ -std=c++17 src/*.cpp -o main
+	g++ -std=c++17 src/*.cpp -Werror -Iinc -o main
```
Comment (DamiXioR) [inline Makefile]: you need to use more alerts

**Review by DamiXioR — COMMENTED** [review]
some ideas

File: src/functions.cpp (hunk 1 above)
Comment (MechaniQ1) [reply src/functions.cpp]: I forgot about it :D

File: src/functions.cpp (hunk 2 above)
Comment (MechaniQ1) [reply src/functions.cpp]: I didn't change anything, it was int from the beginning

Comment (danielzwierz) [reply src/functions.cpp]: it is a part of homework, add alert flags first then watch what happend

Comment (ziobron) [reply src/functions.cpp]: Yup, try to compile this code now with `make`. It should not compile.

**Review by ziobron — CHANGES_REQUESTED** [review]

File: src/functions.cpp (hunk 2 above)
Comment (MechaniQ1) [reply src/functions.cpp]: Right, my bad. It should work now. Thanks! :D

**Comment by ziobron** [comment]
+4 XP Narzędzia#2 – vimtutor
+5 XP Narzędzia#2 – git - konflikty
+4 XP Narzędzia#2 – flagi kompilacji
+3 XP Narzędzia#2 – git - cherrypick
+8 XP delivery before 07.06.2020

{ "requirement_ids": [1119, 1999, 1119, 2000, 1119, 2001, 1119, 2002], "points": [2, 4, 2, 5, 2, 4, 2, 3] }

**Review by ziobron — APPROVED** [review]

**Pull request closed without merging**